lang/sk.json

```diff
@@ -145,141 +145,143 @@
-		"fileNameNotEmpty": "[MISSING]The file name cannot be empty.",
+		"fileNameNotEmpty": "Názov súboru nesmie byť bez znakov.",
 		"folderInvalidCharacters": "Adresár nesmie obsahovať žiadny z nasledujúcich znakov: {disallowedCharacters}",
 		"incorrectExtension": "V tomto adresári nie je povolený tento formát súboru.",
-		"missingFile": "[MISSING]The requested file is no longer available.",
-		"missingFolder": "[MISSING]The folder you are trying to reach is no longer available.",
+		"missingFile": "Súbor už nie je k dispozícii.",
+		"missingFolder": "Priečinok už nie je k dispozícii.",
 		"noUploadFolderSelected": "Pred náhrávaním zvoľte adresár, prosím",
-		"operationCompleted": "[MISSING]Operation completed with errors.",
+		"operationCompleted": "Úloha dokončená s chybami.",
 		"unknown": "Server nemohol dokončiť spracovanie požiadavky. (Chyba {number})",
 		"unknownUploadError": "V priebehu posielania súboru sa vyskytla chyba.",
-		"uploadErrors": "[MISSING]Upload finished with errors.",
+		"uploadErrors": "Nahrávanie ukončené s chybami.",
 		"uploadPermissions": "Nahratie súboru nie je povolené."
 	},
 	"chooseResizedImage": {
-		"title": "[MISSING]Choose Resized",
-		"originalSize": "[MISSING]Original Size",
+		"title": "Zmeniť veľkosť",
+		"originalSize": "Pôvodná veľkosť",
 		"sizes": {
-			"custom": "[MISSING]Custom",
-			"large": "[MISSING]Large",
-			"max": "[MISSING]Max",
-			"medium": "[MISSING]Medium",
-			"small": "[MISSING]Small"
+			"custom": "Vlastné",
+			"large": "Veĺké",
+			"max": "Maximum",
+			"medium": "Stredné",
+			"small": "Malé"
 		}
 	},
 	"editImage": {
-		"adjust": "[MISSING]Adjust",
-		"apply": "[MISSING]Apply",
-		"confirmExit": "[MISSING]Are you sure you want to close? You have unsaved changes to the image.",
-		"crop": "[MISSING]Crop",
-		"downloadAction": "[MISSING]Downloading original image.",
-		"keepAspectRatio": "[MISSING]Keep Aspect Ratio",
-		"loading": "[MISSING]Loading image",
-		"presets": "[MISSING]Presets",
-		"reset": "[MISSING]Reset",
-		"resize": "[MISSING]Resize",
-		"rotate": "[MISSING]Rotate",
-		"rotateAntiClockwise": "[MISSING]90&deg; Left",
-		"rotateClockwise": "[MISSING]90&deg; Right",
-		"save": "[MISSING]Save",
-		"saveDialogFileExists": "[MISSING]File with the same name already exists in folder.",
-		"saveDialogOverwrite": "[MISSING]Overwrite File",
-		"saveDialogSaveAs": "[MISSING]Save as:",
-		"saveDialogTitle": "[MISSING]Save Changes",
-		"title": "[MISSING]Edit Image",
-		"transformationAction": "[MISSING]Applying transformations.",
-		"uploadAction": "[MISSING]Uploading edited image.",
+		"adjust": "Prispôsobiť",
+		"apply": "Použiť",
+		"confirmExit": "Naozaj chcete ukončiť program? Vašé zmeny nebudú uložené.",
+		"crop": "Orezať",
+		"downloadAction": "Sťahujem originálny súbor.",
+		"keepAspectRatio": "Udržať polomer",
+		"loading": "Nahrávam obrázok",
+		"presets": "Zmeniť",
```

Comment:
"Presets" is a noun here, these are pre-defined settings/configurations for image editor filters.